checkstyle-tester/latest-projects-to-test-on.properties

```diff
@@ -0,0 +1,28 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+
+guava|git|https://github.com/google/guava|v28.2||
```

Comment:
Why don't we just remove the specific versions from all of them and have them always be the latest?

Do we want to add this as a feature to the groovy script to ignore commit/tag?

Reply:
> Guava still specifies a specific version.

Mistake, fixed.

> Do we want to add this as a feature to the groovy script to ignore commit/tag?

I would rather not do this, for a few reasons:
1. I would rather get by with current groovy script and put future effort into a whole new system
2. This still does not help us for check regression report in github CI, which is probably how 98% of reports are generated these days. 

We could add some options to github action that would translate to groovy script execution, but I would prefer to not complicate that more either.

Reply:
Generating new reports at  https://github.com/checkstyle/checkstyle/pull/11805, due to missed tag at https://github.com/checkstyle/contribution/pull/613#discussion_r912063874